packages/docs/docs/auth/smart-scopes.md

```diff
@@ -0,0 +1,31 @@
+# SMART Scopes for Access Control
+
+SMART on FHIR’s authorization scheme uses OAuth scopes to communicate (and negotiate) access requirements. There is relevant [background information on HL7.org](http://hl7.org/fhir/smart-app-launch/STU2/scopes-and-launch-context.html).
```

Comment:
I think backing up a few steps and giving some more framing would help make this document more accessible. To me, the biggest thing that's missing is information about "when do I need to use this?". Some questions that would be helpful to answer:

- What is the overall purpose of SMART on FHIR scopes? Might be able to copy some verbiage from [here](http://hl7.org/fhir/smart-app-launch/STU2/app-launch.html)
- What kind of apps will _need_ to use this? Some examples: 
  -  Any 1st party app that Medplum customer builds?
  - An app that launches from inside the Medplum App?
  - An app that I (medplum customer) am building to launch from a 3rd party EPIC instance? 
  - Allowing 3rd party developers to build apps on top of my (medplum customer's) data?

Reply:
Yeah... This page is the best explanation: https://hl7.org/fhir/smart-app-launch/scopes-and-launch-context.html

We should either reference that page directly, or mimic the flow.